src/containers/genePage/genomeFeatureViewer.js

```diff
@@ -79,45 +82,32 @@ class GenomeFeatureViewer extends Component {
   componentDidUpdate() {
   }
 
-  isCodingType() {
-    let proteinCodingTypes = [
-      'gene'
-      , 'protein_coding_gene'
-      , 'protein_coding'
-      , 'ORF'
-    ];
-    return proteinCodingTypes.indexOf(this.props.biotype) >= 0;
-  }
-
   loadData() {
-    if (!this.isCodingType()) {
-      this.setState({loadState: 'noncoding'});
-    }
-    else {
-      this.setState({loadState: 'loading'});
-      fetch(this.trackDataUrl)
-        .then(function (response) {
-          if (!response.ok) {
-            throw Error(response.statusText);
-          }
-          return response;
-        })
-        .then((response) => {
-          response.json().then(data => {
-            this.setState({
-              loadState: 'loaded'
-              , loadedData: data
-            });
-            return data;
-          });
-        })
-        .catch(() => {
-          // console.log(error);
+    this.setState({loadState: 'loading'});
+    this.transcriptTypes = getTranscriptTypes();
```

Comment:
A minor thing here. If you need to work with constants in a React component, you don't have to make it an attribute of the component and define it in lifecycle. (Things that are neither prop or state is used very sparingly in React.) Simply define it before your class definition should work.